triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/simulation/process/ReportingProcess.scala

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf.engine.trigger.simulation
+package process
+
+import akka.actor.typed.{ActorRef, Behavior, PostStop}
+import akka.actor.typed.scaladsl.Behaviors
+import com.daml.ledger.api.v1.command_submission_service.SubmitRequest
+import com.daml.lf.data.Ref.Identifier
+import com.daml.lf.engine.trigger.simulation.TriggerMultiProcessSimulation.TriggerSimulationConfig
+import com.google.rpc.status.{Status => GrpcStatus}
+
+import java.nio.file.{Files, Paths}
+import java.util.UUID
+import scala.collection.immutable.TreeMap
+
+private[simulation] object ReportingProcess {
```

Comment:
The reporting process is responsible for collecting actor system data and writing it to the file system